Makefile

```diff
@@ -139,7 +158,7 @@ fluentd:
 image-cleanup-test-images:
 	cd misc/image-cleanup-test-images; $(MAKE) $(MFLAGS)
 
-get-deps:
+get-deps: get-cni-sources
```

Comment:
True but we have decoupled the `get-cni-sources` from the build which is why I wanted this. 
Everybody should be able to just use get-deps and build the agent image.